bin/install_awscli.sh

```diff
@@ -6,7 +6,7 @@ chmod u+x $INSTALL_DIR/bin/aws
 
 export PATH=~/vendor/awscli/bin:$PATH
 
-mkdir ~/.aws
+mkdir -p ~/.aws
 
 cat >> ~/.aws/credentials << EOF
```

Comment:
Do we want to overwrite here? To avoid appending multiple times.

> cat > ~/.aws/credentials

Reply:
If credentials are changed in EJSON, wouldn't it make sense to overwrite?

Reply:
The ejson is only decrypted once though at container startup AFAIK.